### DISCUSSION TITLE
당근마켓 인턴 중간 회고 | 정현수 기술 블로그

### DISCUSSION THREAD

**Discussion by giscus** [junghyeonsu/junghyeonsu.dev.comment#21] [Announcements]
# 당근마켓 인턴 중간 회고 | 정현수 기술 블로그

인턴을 하면서 느꼈던 것들과 앞으로의 방향성에 대해서

https://junghyeonsu-dev.vercel.app/posts/Daangn-intern-middle-retrospect

**Comment by irrationnelle** [comment]
공감가는 내용이 많네요. 결론도 멋지고요. 잘 읽고 갑니다~

**Reply by junghyeonsu** [reply]
감사합니다^_^
